tests/css_scss/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -856,7 +856,11 @@ p {
 @function myCalculationFunction($some-number, $anotherNumber, $BigNumber) {
   @return $some-number + $anotherNumber + $BigNumber;
 }
-@function AnotherMyCalculationFunction($some-number, $anotherNumber, $BigNumber: 100px) {
+@function AnotherMyCalculationFunction(
+    $some-number,
+    $anotherNumber,
+    $BigNumber: 100px
+  ) {
```

Comment:
Should this be:

```scss
@function AnotherMyCalculationFunction(
  $some-number,
  $anotherNumber,
  $BigNumber: 100px
) {
  @return $some-number + $anotherNumber + $BigNumber;
}
```

?

Reply:
(i.e. remove a space from the beginning of the parameter lines)

Reply:
@lydell yep, first all at rules and tests, second formatting with feedback

Reply:
it is still WIP :smile: thanks for feedback, i ping you when all looks good and tested